Marlin/src/feature/leds/neopixel.cpp

```diff
@@ -44,15 +44,14 @@ void setup_neopixel() {
   pixels.show(); // initialize to all off
 
   #if ENABLED(NEOPIXEL_STARTUP_TEST)
-    delay(2000);
     set_neopixel_color(pixels.Color(255, 0, 0, 0));  // red
-    delay(2000);
+    delay(1000);
     set_neopixel_color(pixels.Color(0, 255, 0, 0));  // green
```

Comment:
Even if the delays are much shorter now - safe_delay() is the function made for delays longer than a few ms.